global_helpers/panther_os_helpers.yml

```diff
@@ -1,4 +1,5 @@
 AnalysisType: global
-Filename: panther.py
+GlobalID: panther_os_helpers
```

Comment:
It should be `oss` since `os` might get confusing with `operating system`

Reply:
`operating system` is literally what I thought at first

Reply:
updated